orchestrator/agent/src/main/kotlin/fr/rob/orchestrator/agent/node/NodeAgentAdapterInterface.kt

```diff
@@ -0,0 +1,6 @@
+package fr.rob.orchestrator.agent.node
+
+interface NodeAgentAdapterInterface {
```

Comment:
We could also use something like NewInstanceRequestInterface (one command = one class) instead of one class for every actions